types/haxball-headless/index.d.ts

```diff
@@ -0,0 +1,110 @@
+// Type definitions for non-npm package haxball-headless-browser 0.1
```

Comment:
This says `haxball-headless-browser` but folder is named `haxball-headless` Please fix the folder name.

Reply:
Done!